packages/components/src/support/Story.tsx

```diff
@@ -11,17 +13,17 @@ const Wrapper = styled.div`
     color: ${({ theme }) => theme.textDefault};
 `;
 
-export const StoryWrapper = (story: any) => (
-    <>
-        <ThemeProvider theme={{ ...intermediaryTheme.light, variant: 'light' }}>
-            <Wrapper>{story.children}</Wrapper>
-        </ThemeProvider>
-
-        <ThemeProvider theme={{ ...intermediaryTheme.dark, variant: 'dark' }}>
-            <Wrapper>{story.children}</Wrapper>
-        </ThemeProvider>
-    </>
-);
+export const StoryWrapper = (story: any) => 
+    React.createElement(Fragment, null,
```

Comment:
Storybook had troubles running the JSX so I did this, I am not sure if it is a porlbme or not, I can tweak it to eat the JS...